packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -606,7 +616,7 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             label: CommonCommands.COLLAPSE_PANEL.label,
             order: '5'
         });
-        registry.registerMenuAction(SHELL_TABBAR_CONTEXT_SPLIT, {
+        registry.registerMenuAction(CommonMenus.VIEW_APPEARANCE_SUBMENU_SCREEN, {
```

Comment:
This shouldn't replace the current registration: it should be added so that it appears in both menus (appearance submenu and tabbar context menu)

Reply:
Ok, so your saying that the `commandId`, `label` and `order` of the `VIEW_APPEARANCE_SUBMENU_SCREEN`  should also be present under `SHELL_TABBAR_CONTEXT_SPLIT`?

Reply:
I mean that there should be two `.registerMenuAction` calls: one that registers the command in the `SHELL_TABBAR_CONTEXT_SPLIT` and one that registers the command for the `CommonMenus.VIEW_APPEARANCE_SUBMENU_SCREEN`

Reply:
Unless I've misunderstood and `CommonMenus.VIEW_APPEARANCE_SUBMENU_SCREEN` is already a submenu of the `SHELL_TABBAR_CONTEXT_SPLIT` menu?

Reply:
@colin-grant-work

Reply:
@Archie27376, I saw your comment and replied - have you made any changes or is what I said unclear?